### ISSUE TITLE
NVM instructions lacking steps

### ISSUE THREAD

**Issue by brookr** [codefellows/setup-guide#53]
Following the [NVM setup instructions](https://github.com/codefellows/setup-guide/blob/master/system-setup/4-node.md), there is no guidance given about the steps that NVM reports need to happen after installation:

```
Add the following to /Users/brookr/.bash_profile or your desired shell
configuration file:
 export NVM_DIR=“$HOME/.nvm”
 [ -s “/usr/local/opt/nvm/nvm.sh” ] && . “/usr/local/opt/nvm/nvm.sh” # This loads nvm
 [ -s “/usr/local/opt/nvm/etc/bash_completion.d/nvm” ] && . “/usr/local/opt/nvm/etc/bash_completion.d/nvm” # This loads nvm bash_completion
```

We should be telling students how to take action on this. What have they been doing up to now? It doesn't seem to work unless this step is done.

**Comment by brookr** [comment]
Maybe this is being handled in the environment setup step?

`echo 'source $(brew --prefix nvm)/nvm.sh' >> ~/.profile`

I'm really not clear how that command even works as a part of Step 1, before NVM is installed...

**Comment by RogerHuba** [comment]
Fixed with PR #63
